packages/serverpod/lib/src/server/log_manager/log_manager.dart

```diff
@@ -424,57 +395,40 @@ class LogManager {
     }
     return null;
   }
+}
 
-  /// Returns a list of logs for all open sessions.
-  List<SessionLogInfo> getOpenSessionLogs(
```

Comment:
Question: This means that we are removing support for this feature now, in insights now right? Should probably be noted in the description.

Reply:
Yes indeed, I'm not sure if we use it, but discussed this with victor a week or so back and it seems like a rather redundant feature, hence removing it.